src/bridgeService.ts

```diff
@@ -481,13 +480,11 @@ export class BridgeService {
       const accessoryUUID = uuid.generate(accessoryType + ":" + (uuidBase || displayName));
       const accessory = new Accessory(displayName, accessoryUUID);
 
-      // listen for the identify event if the accessory instance has defined an identify() method
       if (accessoryInstance.identify) {
+        // listen for the identify event if the accessory instance has defined an identify() method
         accessory.on(AccessoryEventTypes.IDENTIFY, (paired: boolean, callback: VoidCallback) => {
-          // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-          // @ts-ignore
           // eslint-disable-next-line @typescript-eslint/no-empty-function
-          accessoryInstance.identify!(() => { }); // empty callback for backwards compatibility
+          accessoryInstance.identify?.(() => { }); // empty callback for backwards compatibility
```

Comment:
Technically, `accessoryInstance.identify` can be set to `null | undefined` after initialization and therefor this will throw as it is not a function. Safeguarding.